<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:grey;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSalvar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
background-color:green;</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCPF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="placeholderText">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNome">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="placeholderText">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBEstado">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBCidade">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTelefone">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="placeholderText">
     <string>Telefone</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
